Sources/Basic/OutputByteStream.swift

```diff
@@ -230,6 +237,51 @@ public class _OutputByteStreamBase: OutputByteStream {
         }
         buffer += bytes
     }
+
+    /// Write an ArraySlice of bytes to the buffer.
+    public final func write(_ bytes: ArraySlice<UInt8>, buffered: Bool = true) {
```

Comment:
can't we use @_specialize or something instead of copying the loop?

Reply:
I’m not familiar with that annotation.  If it does what I infer, it could make sense combined with making “unbuffered” a general attribute on the stream.